### DISCUSSION TITLE
About dividend

### DISCUSSION THREAD

**Discussion by jackysywk** [ssantoshp/Empyrial#73] [General]
May I ask whether the dividend is included in the return?

**Comment by santoshlite** [comment]
Yes, we use the Adj. close price which does include dividends!

**Comment by timjn** [comment]
I am getting this error running previously working Empyrial example code
On google colab


type list doesn't define __round__ method

can anyone help ?

**Reply by santoshlite** [reply]
I think this should be fixed now
